src/main/java/com/jalasoft/sdfc/pages/AppLaunchPage.java

```diff
@@ -0,0 +1,23 @@
+package com.jalasoft.sdfc.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AppLaunchPage extends AbstractPage {
+    private static final String PRIVACY_CSS = "//a[@title='%s']";
+    @FindBy(css = ".slds-icon-waffle_container")
+    private WebElement AppLauncherButton;
```

Comment:
I think this should be in the Header and once it  is clicked the method should return an AppLaunchPage object

Reply:
please review the last commit